### PR TITLE
Add Scroll-driven animations (positive)

### LINKED ISSUE #347: Scroll-driven Animations
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Scroll-driven Animations
* Specification or proposal URL: https://drafts.csswg.org/scroll-animations-1
* Caniuse.com URL (optional): N\A
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1324602
* Mozillians who can provide input (optional): @botond-ballo

### Other information

There appears to be some support for this feature already (given the prior work done on it), but it would good to add an entry for it. Also the spec now contains a declarative API (@scroll-timline etc), which needs evaluation.

(cc @majido, so he knows this exists).

### PR DESCRIPTION
**Pull request by zcorpan**
Closes #347

### CODE REVIEW

**Comment by zcorpan** [comment]
Also see https://github.com/Fyrd/caniuse/pull/6956 for caniuse.

**Review by martinthomson — APPROVED** [review]

**Review by tantek — CHANGES_REQUESTED** [review]
This position reflects our evaluation per the issue.

One minor nit in the description, it should mention CSS properties first (which are in the draft), and drop "markup". I.e.:

s/Defines an API and markup for/Defines CSS properties and an API for

File: activities.json [hunk 1]
```diff
@@ -1371,6 +1371,19 @@
     "title": "Screen Wake Lock API",
     "url": "https://w3c.github.io/wake-lock/"
   },
+  {
+    "ciuName": null,
+    "description": "Defines an API and markup for creating animations that are tied to the scroll offset of a scroll container.",
```
Comment (zcorpan) [inline activities.json:1376]:
```suggestion
    "description": "Defines CSS properties and an API for creating animations that are tied to the scroll offset of a scroll container.",
```

**Comment by zcorpan** [comment]
@tantek fixed, and [suggested that change to the spec](https://github.com/w3c/csswg-drafts/pull/9883).

**Review by tantek — APPROVED** [review]
Thank you and nicely done with the upstream spec fix as well!

**Pull request merged**